test_env_vars.yaml

```diff
@@ -118,3 +118,12 @@ various_packages:
     repo: https://github.com/akhmanova/cachito-gomod-test
     ref: 1827221d787cbd1e979b339cfbbf59728eddf0d4
     dependencies_count: 7
+# Test data for pip packages
+pip_packages:
+  without_deps:
+    repo: https://github.com/akhmanova/cachito-pip-without-deps.git
+    ref: fb07c8492432631d557aba15e1715af9cef9c844
+    expected_files:
```

Comment:
cn we add comments to document this? In special, it would be nice to make it explicit that there must be URLs to the expected files.